encryption_works.md

```diff
@@ -275,7 +275,7 @@ Thanks to the Snowden documents, we know that for many Internet users, powerful
 
 According to leaked NSA slides with titles such as ["Tor Stinks"](http://www.theguardian.com/world/interactive/2013/oct/04/tor-stinks-nsa-presentation-document), Tor is "still the King of high secure, low latency Internet Anonymity" with "no contenders." However, the same slides also indicate that the surveillance agency can de-anonymize "a very small fraction" of Tor users. Also important to note is that these slides are also several years old now, so we don't know what the current status of these powerful organization's capabilities might be.
 
-Such an attack attack like this becomes much harder as the number of nodes and the volume of traffic increases -- not impossible, but more time-consuming and expensive. Everyone can pitch in and bolster the Tor network by setting up a node; since colleges and universities are often their own ISP with strong infrastructure, students and staff are particularly [encouraged](https://www.eff.org/torchallenge/tor-on-campus.html) to set up a Tor node on campus.
+Such an attack attack like this becomes much harder as the number of nodes and the volume of traffic increases -- not impossible, but more time-consuming and expensive. Everyone can pitch in and bolster the Tor network by setting up a node, which Tor traffic can flow through. Since colleges and universities often have strong ISP infrastructure, students and staff are particularly [encouraged](https://www.eff.org/torchallenge/tor-on-campus.html) to set up a Tor node on campus. (If you're concerned about the legality of running a Tor exit node, check out the Tor Project's [legal FAQ](https://www.torproject.org/eff/tor-legal-faq.html.en).)
```

Comment:
"Such an attack like this" is redundant-- "Attacks like these"?  "Such an attack"?  (also, dupe of "attack" in the sentence.)